rest/db-file-get.rst

```diff
@@ -1,10 +1,27 @@
 GET /rest/db/file
 =================
 
-Returns most data available about a given file, including version and
-availability. Takes ``folder`` and ``file`` parameters. ``local`` and
-``global`` refer to the current file on disk and the globally newest file,
-respectively.
+Returns data about a given file, including version and
+availability.
+
+Request
+-------
+
+The call requires parameters for `folder` and `file` in the querystring:
+
+- `folder` is the folder code which you can find in the synthing UI, e.g. `5camp-slpa8`
+
+- `file` is the full path to the file you are interested in. The path and filename must be correctly url-encoded.
```

Comment:
To me "full path" indicates an absolute file path, while the path here is relative to the folder root path. And seems confusing to mention "path and filename", as if they were somehow separate, I'd just say `The path must ...`.

Reply:
Good point, alternate wording added in 89ee8d1ab11b5bb1fe9fcf5b3daac1211d9af55e